MvvmCross_Forms_Examples.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
```

Comment:
Thanks Martijn
My current laptop running VS2013 is broken. Will download VS 2013 from subscriber downloads and set up on a VM. SLN file is very simple so if we are lucky simply changing it to 2013 should work. The real danger would be in VS2013 running older versions of NuSpec etc., but what we have here should be OK.

VS2015 RTM will be available from MSDN subscriber downloads July / 20 and presumably to the public then or shortly after. Presumably you get a free trial period typically 180 days if you do not have MSDN.